cypress/e2e/tests/rbac/migrator.test.ts

```diff
@@ -59,16 +75,6 @@ describe(["@tier2", "@rhsso"], "Migrator RBAC operations", () => {
         Application.validateCreateAppButton(this.rbacRules);
     });
 
-    it("Migrator, validate assess application button", function () {
-        //Migrator is not allowed to create applications
-        application.validateAssessButton(this.rbacRules);
-    });
-
-    it("Migrator, validate review application button", function () {
-        //Migrator is not allowed to review applications
-        application.validateReviewButton(this.rbacRules);
-    });
-
     it("Migrator, validate presence of import and manage imports", function () {
```

Comment:
change the test name as well to cover all buttons.

Reply:
Done